src_app/test_scenes/sceneModelAnimated.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import { Factory } from 'magic-script-components';
 
 class SceneModelAnimated extends React.Component {
   render () {
+    const path = (Platform.OS === 'Android') ? require('../../resources/hedra_anim.glb') : require('../../resources/hedra_anim.glb')
```

Comment:
Please do not change `sceneModelAnimated.js` in this PR. There is already another branch with changes to models scenes waiting for `Platform` support.